tests/input/example-class.php

```diff
@@ -23,6 +23,7 @@ class Example implements \IteratorAggregate
 {
     private const VERSION = \PHP_VERSION - (PHP_MINOR_VERSION * 100) - PHP_PATCH_VERSION;
 
+
```

Comment:
FYI currently there is no sniff included that fixes the extra lines inside a class.. it does fix the extra lines between functions but it does not look inside the whole class.

Reply:
SlevomatCodingStandard.Classes.ClassMemberSpacing

Reply:
Great.. will create a new PR with that..

Reply:
There are also:
- `MethodSpacing`
- `PropertySpacing`
- `ConstantSpacing`